### DISCUSSION TITLE
Settings: Examples for syntax of "Path or name of additional note(s) containing sorting spec"

### DISCUSSION THREAD

**Discussion by SlackerJack** [SebastianMC/obsidian-custom-sort#77] [Q&A] [answered]
I have 'inbox' and that seems to work as I'd expect, but if I add 'index' I get the FAILED... Tried multiple delimiters (space, comma, semicolon) 

Any examples or document to help clarify this a bit?

Also, the setting 'name' being more of a description than a name makes it difficult to search for info. Not sure if there is simple way to help that.

Thank you!

**Comment by SebastianMC** [comment, marked as answer]
Hi @SlackerJack,

if I get it correctly, you're trying to enter both 'inbox' and 'index' in the settings. This has no chance of success by definition, the setting accepts only a single value - you can enter either 'inbox' or 'index', not both

If you find it useful to be able to enter multiple values there, please create new issue in [issues](https://github.com/SebastianMC/obsidian-custom-sort/issues). I'm working now on a major update of the plugin and after that the extension could be done as well, at a glance it seems to be a low hanging fruit

thanks for your input!

**Reply by SlackerJack** [reply]
Ah! I read it wrong. I saw the note(s) part in the description and assumed it meant more than one note name. My bad. 

I worked around what I was trying to do with Folder Notes. The combo seems to be doing exactly what I needed. So, unless others would find this useful I don't think we need to add more work to your plate :)

Thank you for the quick reply. And thank you for your work on this plugin. So Useful!
